api/health/v1/health.proto

```diff
@@ -4,7 +4,7 @@ package kessel.relations.v1;
 
 import "google/api/annotations.proto";
 
-option go_package = "ciam-rebac/api/health/v1;v1";
+option go_package = "relations-api/api/health/v1;v1";
```

Comment:
Does this go_package resolve properly?

See:

- https://stackoverflow.com/questions/61666805/correct-format-of-protoc-go-package
- https://protobuf.dev/reference/go/go-generated/#package

Reply:
Proto docs also suggest we should drop the `;v1` part

Reply:
1) renaming package: to full import path (URL in the browser - after relationsa-pi - get the dir 
ex: github.com/project-kessel/relations-api/api/health/v1 

Make the package name and go_package the same

2) remove ;v1 and check  - deprecated as per: https://protobuf.dev/reference/go/go-generated/#package (if hugely disruptive - defer this - create a ticket for future)